linter.sh

```diff
@@ -1,28 +1,17 @@
 #!/usr/bin/env bash
 
-if ! [ -d $GOPATH/src/github.com/alecthomas/gometalinter/  ]; then
-    echo "downloading gometalinter"
-    go get -u github.com/alecthomas/gometalinter
-    gometalinter --install
-    if [  "$?" != "0" ] ; then
-        echo "error when installing gometalinter"
-    exit 1
-    fi
-fi
-
-gometalinter ./... 2>&1 > gometalinter-err.out
-
+if ! [ -d $GOPATH/src/github.com/golangci/golangci-lint/  ]; then
+    echo "downloading golangci-lint"
+    go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
I think we should use:
```
curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
```

Since it says this is the recommended way here to have a stable version: https://github.com/golangci/golangci-lint#ci-installation.

And in that case the above if check would be: `if ! command -v golangci-lint > /dev/null; then`